nextcord/http.py

```diff
@@ -2337,3 +2338,88 @@ def get_event_users(
             event_id=event_id,
         )
         return self.request(r, params=params)
+
+    def list_guild_auto_moderation_rules(
+        self, guild_id: Snowflake
+    ) -> Response[List[auto_moderation.AutoModerationRule]]:
+        r = Route("GET", "/guilds/{guild_id}/auto-moderation/rules", guild_id=guild_id)
+        return self.request(r)
+
+    def get_auto_moderation_rule(
+        self,
+        guild_id: Snowflake,
+        auto_moderation_rule_id: Snowflake,
+    ) -> Response[auto_moderation.AutoModerationRule]:
+        r = Route(
+            "GET",
+            "/guilds/{guild_id}/auto-moderation/rules/{auto_moderation_rule_id}",
+            guild_id=guild_id,
+            auto_moderation_rule_id=auto_moderation_rule_id,
+        )
+        return self.request(r)
+
+    def create_auto_moderation_rule(
+        self,
+        guild_id: Snowflake,
+        data: auto_moderation.AutoModerationRuleCreate,
+        *,
+        reason: Optional[str] = None,
+    ) -> Response[auto_moderation.AutoModerationRule]:
+        valid_keys = (
```

Comment:
Could we put both of these `valid_keys` into one location?

Reply:
`trigger_type` is not included in `modify_`